LoopKitUI/Views/Settings Editors/DeliveryLimitsEditor.swift

```diff
@@ -379,13 +392,39 @@ public struct DeliveryLimitsEditor: View {
                 )
             )
         case .saveError(let error):
-            return Alert(
-                title: Text(LocalizedString("Unable to Save", comment: "Alert title when error occurs while saving a schedule")),
-                message: Text(error.localizedDescription)
-            )
+            switch error {
+            case SaveTherapySettingsError.cancelTempBasalError(let error):
+                return cancelTempBasalAlert(error)
+            default:
+                return SwiftUI.Alert(
+                    title: Text(LocalizedString("Unable to Save", comment: "Alert title when error occurs while saving a schedule")),
+                    message: Text(error.localizedDescription)
+                )
+            }
         }
     }
 
+    private func cancelTempBasalAlert(_ error: Error?) -> SwiftUI.Alert {
```

Comment:
There are many reasons why saving the max temp might fail, cancelling basal is just one of them. For the other errors, we'll need pass an `Error` with a sufficient `localizedDescription` (and possibly recovery suggestion). What is the reason this has a special type? Would recommend having a unified handling of errors here, unless there is a specific and clear UX need for this one to be different.

Reply:
There is a specific UX the team wants separate from the save errors (and which already was implemented in v1.1.0).  See [LOOP-3856](https://tidepool.atlassian.net/browse/LOOP-3856) for more details.